apps/dashboard/src/hooks/useTerminalConnection.ts

```
@@ -279,6 +293,70 @@ export function useTerminalConnection({
               }
               requestedResumeRef.current = false;
               restartedAfterResumeFailureRef.current = false;
+              const terminalSnapshot = terminalHostStore.getSnapshot();
```

Biome (2.5.3)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)